### DISCUSSION TITLE
Smart card integration and sshg3

### DISCUSSION THREAD

**Discussion by byrdman1982** [blinksh/blink#1877] [Ideas]
As of iOS 16, users can use a smart card on apple devices (https://support.apple.com/guide/deployment/use-a-smart-card-on-iphone-and-ipad-dep8b8c8927a/web).  I’ve successfully used this feature to connect to web sites.  

I use a smart card to connect to a remote computer using sshg3.  It would be great to have this feature available on blink.  Is there a possibility of building this into blink?

I think this is slightly different that #1875 but maybe somewhat related.

**Comment by carloscabanero** [comment]
I gotta be honest, never heard of sshg3 (https://www.helpnetsecurity.com/2005/02/14/ssh-unveils-ssh-g3-a-powerful-third-generation-secure-shell-architecture-for-ssh-tectia/). It does look like it is full of proprietary implementations, so it would be a minefield for us.

I'm curious though. Do you use smart cards in other regular ssh connections? Is it the same? And what type of machines or systems require sshg3?
